.github/workflows/timed-task.yml

```diff
@@ -0,0 +1,10 @@
+name: Timed Task
+on:
+  schedule:
+    - cron: '0 16 * * *'
+jobs:
+  clean-image:
+    runs-on: aliyun
+    steps:
+      - name: Cleanup image
+        run: docker rmi $(docker images --filter "dangling=true" -q --no-trunc)
```

Comment:
If `docker images --filter "dangling=true" -q --no-trunc` return `empty`, error info `"docker rmi" requires at least 1 argument` will occur. Maybe `docker image prune -f` is enough

Reply:
good suggestion